cabal.project

```diff
@@ -12,10 +12,13 @@ repository cardano-haskell-packages
 
 -- See CONTRIBUTING for some Nix commands you will need to run if you
 -- update either of these.
--- Bump this if you need newer packages from Hackage
-index-state: 2022-12-01T00:00:00Z
--- Bump this if you need newer packages from CHaP
-index-state: cardano-haskell-packages 2022-11-15T00:00:00Z
+-- repeat the index-state for hackage to work around haskell.nix parsing limitation
+index-state: hackage.haskell.org 2022-12-01T00:00:00Z
```

Comment:
From https://github.com/haskell/cabal/issues/8568, it seems to me that we could delete this given the immediately succeeding multi-line `index-state` declaration, right?
```suggestion
```

Reply:
ah damn it, that's supposed to be
```
index-state: 2022-12-01T00:00:00Z
```
too much copy and paste :disappointed:

That line is only useful for haskell.nix, cabal will override it with the following one.